content/docs/reference-react-dom-server.md

```diff
@@ -15,31 +15,31 @@ import ReactDOMServer from 'react-dom/server';
 var ReactDOMServer = require('react-dom/server');
 ```
 
-## Overview {#overview}
+## סקירה כללית {#overview}
 
-The following methods can be used in both the server and browser environments:
+ניתן להשתמש במתודות הבאות הן בסביבת השרת והן בסביבות דפדפן:
 
 - [`renderToString()`](#rendertostring)
 - [`renderToStaticMarkup()`](#rendertostaticmarkup)
 
-These additional methods depend on a package (`stream`) that is **only available on the server**, and won't work in the browser.
+מתודות נוספות אלה תלויות בחבילה (`stream`) אשר **זמינה בשרת בלבד**, ולא יפעלו בדפדפן.
 
 - [`renderToNodeStream()`](#rendertonodestream)
 - [`renderToStaticNodeStream()`](#rendertostaticnodestream)
 
 * * *
 
-## Reference {#reference}
+## סימוכין {#reference}
```

Comment:
I think it'll be too complicated for Hebrew readers, but I understand it has a good meaning. I would like to see what other contributors think about it and by the way, we should also add it to the style guide #22

Reply:
It doesn't fit in every context of the word, but in this context, as I have to translate it as a header, this is the most suitable translation I found. Wouldn't mind changing it later on if we find a better translation.

Reply:
I found some, but I don't think they are a good fit (e.g עיון, התייחסות). What about leaving it as it is?

Reply:
I wouldn't want to keep this in English since it is not a technical term and I find this translation to be accurate. I understand it sounds a little legal-ish, but this is the language :)

Reply:
We have to remember that our job is to make the reader’s life easier and obviously, in this case, it is not. Let's leave it open for now until we find something else.